devp2p/README.md

```diff
@@ -2,9 +2,15 @@
 
 Run a portal network node that you want to test and pass node's Enr as a target node.
 
+## Running tests
+To test a target client against a dummy peer, do:
 ```sh
 cd devp2p
 
-TRIN_DATA_PATH="/tmp/devp2p" RUST_LOG=debug cargo run -p devp2p -- --target_node enr:-IS4QBDHCSMoYoC5UziAwKSyTmMPrhMaEpaE52L8DDAkipqvZQe9fgLy2wVuuEJwO9l1KsYrRoFGCsNjylbd0CDNw60BgmlkgnY0gmlwhMCoXUSJc2VjcDI1NmsxoQJPAZUFErHK1DZYRTLjk3SCNgye9sS-MxoQI-gLiUdwc4N1ZHCCIyk
-
-```
\ No newline at end of file
+./boot.sh
```

Comment:
I think it will be better if we can support cross-client testing (not only `trin` nodes), which makes the changes you propose in the README not viable.

Reply:
I was thinking that we could start by making a testing framework just for trin, and then expand to cross-client, since that would probably be faster. But I'm not sure--what exactly would a cross-client testing framework entail (vs. a non cross client)?